static/js/formatters-internal.js

```diff
@@ -566,3 +566,24 @@ export function getYoutubeUrl(videos = []) {
     : null;
   return youtubeVideoUrl;
 }
+
+/**
+ * construct an array of displayable category names based on given category ids from liveAPI
+ * and a mapping of category ids to names.
+ * 
+ * @param {string[]} categoryIds category ids from liveAPI
+ * @param {Object[]} categoryMap mapping of category ids to names
```

Comment:
I think we want to remove the square brackets here, it gives the appearance of an Array of Objects.

Reply:
Got it!